### PR TITLE
[JIRA MIGRATION] Update inline code fragments transfer rules

### PR DESCRIPTION
**Pull request by aasmolsky**
Fix issue with inline code and code fragment
https://youtrack.softswiss.net/issue/CASINO-16776

Tested on https://youtrack.softswiss.net/issue/PDB-170
Result - https://jira-copy.softswiss.net/browse/REACT-333

### CODE REVIEW

File: lib/syntax_migrator.rb [hunk 1]
```diff
@@ -66,8 +66,8 @@ def migrate_code_blocks(text)
     end
 
     def migrate_one_code_lines(text)
-      text.gsub!('``', '{noformat}')
-      text.gsub!('`', '{noformat}')
+      text.gsub!(/``(.+?)``/, '{{\1}}')
```
Comment (zaptot) [inline lib/syntax_migrator.rb:69]: how it will work with:

\`\`aasdfasf
aasdfasf
\`\`

Comment (aasmolsky) [reply lib/syntax_migrator.rb:69]: It will skip the text and transfer to Jira as it is (with ``). Is it ok?

Comment (zaptot) [reply lib/syntax_migrator.rb:69]: don't think so

Comment (aasmolsky) [reply lib/syntax_migrator.rb]: Updated https://github.com/zaptot/youtrack_to_jira_migrator/pull/12/commits/93fa61768a77b88828e2954d492f728a034534a4
Looks ugly but it works as we've discussed.
WDYT?

**Pull request merged**